database/src/main/java/com/decathlon/ara/repository/ErrorRepository.java

```diff
@@ -39,6 +39,8 @@
     // NO projectId: patterns is already restrained to the correct project
     Page<Error> findDistinctByProblemPatternsInOrderById(List<ProblemPattern> patterns, Pageable pageable);
 
+    Page<Error> findDistinctByProblemPatternsInOrderByExecutedScenarioRunExecutionTestDateTimeDesc(List<ProblemPattern> patterns, Pageable pageable);
```

Comment:
Missing documentation

Reply:
done (cf my comment)

Reply:
Here we're not talking about comments but about documentation 😄 
I got your point, but that's also what reviews are supposed to do, make sure that all the changes are made. 
If the code changes and the documentation doesn't in a PR, we can view it and change it :) 

Here is an example : in your method, what happens if one of the param is null ? And the return is always an object or can't be null ?
Also your method (thanks to SpringJDBC...) has 82 characters in it. 82chars of camelCase can be harsh to read to know a purpose of it, spaces can helps a lot in comprehension 😄